projects/polygon-erc-20s.json

```diff
@@ -67,5 +67,173 @@
+        "contractType"     : "ERC20",
+        "decimals"         : 18,
+        "contractAddress"  : "0xa3Fa99A148fA48D14Ed51d610c367C61876997F1"
+        }, 
+{
+        "assetName"        : "Gains Network",
+        "tokenImage"       : "https://polygonscan.com/token/images/gainsnetwork_32.png"
+        "contractType"     : "ERC20",
+        "decimals"         : 18,
+        "contractAddress"  : "0xE5417Af564e4bFDA1c483642db72007871397896"
+  }, 
+{
+        "assetName"        : "PlanetIX",
+        "tokenImage"       : "https://polygonscan.com/token/images/planetix_32.png"
+        "contractType"     : "ERC20",
+        "decimals"         : 18,
+        "contractAddress"  : "0xE06Bd4F5aAc8D0aA337D13eC88dB6defC6eAEefE"
+        }, 
+{
+        "assetName"        : "EverRise",
+        "tokenImage"       : "https://polygonscan.com/token/images/everrise_32.png"
+        "contractType"     : "ERC20",
+        "decimals"         : 18,
+        "contractAddress"  : "0xC17c30e98541188614dF99239cABD40280810cA3"
+        }, 
+{
+        "assetName"        : "Decentral Games",
+        "tokenImage"       : "https://polygonscan.com/token/images/decentralgame_32.png?v=5"
+        "contractType"     : "ERC20",
+        "decimals"         : 18,
+        "contractAddress"  : "0xef938b6da8576a896f6E0321ef80996F4890f9c4"
+        }, 
+{
+        "assetName"        : "Klima DAO Token",
+        "tokenImage"       : "https://polygonscan.com/token/images/klimadao_32.png"
+        "contractType"     : "ERC20",
+        "decimals"         : 9,
+        "contractAddress"  : "0x4e78011Ce80ee02d2c3e649Fb657E45898257815"
+        },
+ {
+        "assetName"        : "RouteToken",
+        "tokenImage"       : "https://polygonscan.com/token/images/routerprotocol_32.png?=v2"
+        "contractType"     : "ERC20",
+        "decimals"         : 18,
+        "contractAddress"  : "0x16ECCfDbb4eE1A85A33f3A9B21175Cd7Ae753dB4"
+        }, 
+{
+        "assetName"        : "KastaToken",
+        "tokenImage"       : "https://polygonscan.com/token/images/kasta2_32.png"
+        "contractType"     : "ERC20",
+        "decimals"         : 18,
+        "contractAddress"  : "0x235737dBb56e8517391473f7c964DB31fA6ef280"
+        },
+ {
+        "assetName"        : "MANTRA DAO Token",
+        "tokenImage"       : "https://polygonscan.com/token/images/mantradao_32.png"
+        "contractType"     : "ERC20",
+        "decimals"         : 18,
+        "contractAddress"  : "0xC3Ec80343D2bae2F8E680FDADDe7C17E71E114ea"
+        }, 
+{
+        "assetName"        : "Stake DAO Token",
+        "tokenImage"       : "https://polygonscan.com/token/images/stakedaopoly_32.png"
+        "contractType"     : "ERC20",
+        "decimals"         : 18,
+        "contractAddress"  : "0x361A5a4993493cE00f61C32d4EcCA5512b82CE90"
+        }, 
+{
+        "assetName"        : "Decentral Games Governance",
+        "tokenImage"       :.  "https://polygonscan.com/token/images/decentralgamesgov_32.png?v=1"
```

Comment:
This will be approved but there's a typo in line 233:

Remove the "."(dot) wrongly placed before the beginning of the URL.